ui/app/src/components/PublishDialog/PublishDialogContainer.tsx

```diff
@@ -221,34 +218,37 @@ export function PublishDialogContainer(props: PublishDialogContainerProps) {
 			})
 				.pipe(
 					switchMap((dependenciesByType) => {
-						const dependencies = [...dependenciesByType.hardDependencies, ...dependenciesByType.softDependencies];
+						const dependencies = [
+							...dependenciesByType.hardDependencies.map((dep) => dep.path),
+							...dependenciesByType.softDependencies.map((dep) => dep.path)
+						];
 						if (dependencies.length) {
-							return fetchDetailedItems(siteId, dependencies).pipe(
-								map((detailedItemsList) => {
-									return { dependenciesByType, detailedItemsList };
+							return fetchContentItems(siteId, dependencies).pipe(
```

Comment:
@rart unfortunately I wasn't able to remove this fetch as we discussed, since there are a couple of utils used in here that require ContentItems:
- getDateScheduled (retrieves the field from `live` or `staging` prop)
- getComputedPublishingTarget (gets the value from `stateMap`)